sdk/search/azure-search-documents/src/samples/java/com/azure/search/documents/SearchSuggestionExample.java

```diff
@@ -70,7 +70,7 @@ private static void suggestWithFuzzySearch(SearchClient searchClient) {
             .setUseFuzzyMatching(true);
 
         PagedIterableBase<SuggestResult, SuggestPagedResponse> suggestResult =
-            searchClient.suggest("hitel", "sg", suggestOptions, Context.NONE);
+            searchClient.suggest("hotel", "sg", suggestOptions, Context.NONE);
```

Comment:
This was specifically `hitel` to show off fuzzy matching capabilities with the suggester